cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/validation/SidecarValidation.java

```diff
@@ -0,0 +1,24 @@
+package org.apache.cassandra.spark.validation;
+
+import org.apache.cassandra.sidecar.client.SidecarClient;
+import org.apache.cassandra.sidecar.common.data.HealthResponse;
+
+public class SidecarValidation implements StartupValidation
+{
+    private final SidecarClient sidecar;
+
+    public SidecarValidation(SidecarClient sidecar)
+    {
+        this.sidecar = sidecar;
+    }
+
+    @Override
+    public void validate()
+    {
+        HealthResponse health = sidecar.sidecarHealth().get();
```

Comment:
should we get with a timeout? you don't want to block indefinitely

Reply:
Good point, added timeouts both here and in `CassandraValidation`.